elasticsearch/scripts/createIndex.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+#
+# To create a index in elasticsearch
+#
+
+# shards & replicas are default values.
+#
+
+TODAY=$(date '+%Y%m%d')
+INDEX="publiccode_$TODAY"
+
+generate_index_settings() {
+  cat <<EOF
+{
+  "settings" : {
+    "index" : {
+      "number_of_shards" : 5,
+      "number_of_replicas" : 1
+    }
+  },
+  "mappings": {
+    "software": {
```

Comment:
can we reload the definition from elasticsearch/mappings/software.json instead of replicate it here?